Description: Disable object and std-object featuresets
 rust-object depends on an obsolete version of rust-goblin.
uthor: Peter Michael Green <plugwash@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-04-13

--- rust-addr2line-0.10.0.orig/Cargo.toml
+++ rust-addr2line-0.10.0/Cargo.toml
@@ -55,10 +55,10 @@ default-features = false
 [dependencies.lazycell]
 version = "1.0"
 
-[dependencies.object]
-version = "0.12"
-optional = true
-default-features = false
+#[dependencies.object]
+#version = "0.12"
+#optional = true
+#default-features = false
 
 [dependencies.rustc-demangle]
 version = "0.1"
@@ -83,8 +83,8 @@ version = "0.3"
 
 [features]
 alloc = ["gimli/alloc"]
-default = ["rustc-demangle", "cpp_demangle", "std-object"]
+default = ["rustc-demangle", "cpp_demangle"]
 std = ["gimli/std", "intervaltree/std", "smallvec/write"]
-std-object = ["std", "object", "object/std"]
+#std-object = ["std", "object", "object/std"]
 [badges.travis-ci]
 repository = "gimli-rs/addr2line"
